recipe/meta.yaml

```
@@ -28,9 +28,12 @@ source:
     - patches/do-not-make-symlinks-on-windows.patch
 
 build:
-  number: 7
+  number: 8
   # string: aster_py{{ CONDA_PY }}_mpi_{{ mpi }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [mpi != "nompi"]
   # string: aster_py{{ CONDA_PY }}_nompi_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}      # [mpi == "nompi"]
+  run_exports:
+    - zlib
+    - hdf5
```

```suggestion
    - {{ pin_subpackage(name, 'x.x') }}
```

I'm just guessing at the ABI guarantees here.

HDF5 and ZLIB will be pulled in because they declare their own "run_exports". You need to declare that:

* If a packages needed libmed at compile time.
* It will need a libmed with the same major and minor, but possibly larger patch version, at runtime.